deploy/probers/README.md

```diff
@@ -0,0 +1,69 @@
+# Writing your own prober
+
+Thanks for helping make Studio a more stable application!
+
+Probers are small programs that check that certain functionality in live
+production is working. Probers shouldn't be complex and require more than a
+hundred lines of code. That might be a sign that your prober can be broken down
+into multiple smaller probers, or that it should be an integration test instead.
```

Comment:
For that language about what probes shouldn't do, maybe something like "In our current setup, probers shouldn't do more than sanity check that a particular page, or API endpoint, is functioning. Testing for things like proper validation, API responses, or other checks should be handled by unit or integration tests. A good litmus test is if you need to set up more than a trivial amount of test data or find yourself importing more than one or two functions from the app or test suite, you're probably crossing into test territory. When writing and reviewing probes, please ensure they are kept small and simple, and also lightweight as they must run frequently."